.github/workflows/release.yml

```diff
@@ -3,42 +3,100 @@ on:
   push:
     branches:
       - master
+  # TODO: uncomment before merging
+  pull_request:
 
 jobs:
-  stable:
-    uses: the-guild-org/shared-config/.github/workflows/release-stable.yml@main
-    with:
-      releaseScript: release
-      nodeVersion: 18
-      packageManagerVersion: modern
-    secrets:
-      githubToken: ${{ secrets.GITHUB_TOKEN }}
-      npmToken: ${{ secrets.NODE_AUTH_TOKEN }}
-
-  docker-image:
-    needs: stable
-    if: needs.stable.outputs.published == 'true'
+  # TODO: uncomment before merging
+  # stable:
+  #   uses: the-guild-org/shared-config/.github/workflows/release-stable.yml@main
+  #   with:
+  #     releaseScript: release
+  #     nodeVersion: 18
+  #     packageManagerVersion: modern
+  #   secrets:
+  #     githubToken: ${{ secrets.GITHUB_TOKEN }}
+  #     npmToken: ${{ secrets.NODE_AUTH_TOKEN }}
+
+  serve-with-hive-docker-image:
```

Comment:
This step should be named just as `docker-build`. the HCL is in charge of building different variants.